docs/create_equinix_cluster.md

```diff
@@ -0,0 +1,109 @@
+# Create Equinix Metal cluster using CAPI / CAPEM
+
+## Overview
+
+- We are using machines contributed to the CNCF by Equinix Metal.
+- These steps manually create a cluster using CAPI / CAPEM using a local
+Kind cluster for the management cluster.
+- These docs are based on the Equinix CAPI [guide](https://deploy.equinix.com/developers/guides/kubernetes-cluster-api/).
+- Using [Podman](https://github.com/kubernetes-sigs/cluster-api-provider-packet/issues/496)
+is recommended by Equinix for MacOS.
+- We plan to automate these steps later using GitHub Actions and an IaC tool.
+
+## Create Cluster
+
+- Log in to the Equinix Metal [console](https://console.equinix.com/) and get
+the project API key from the project settings. If it doesn't exist create it.
+
+```sh
+export PACKET_API_KEY="<PROJECT_API_KEY>"
+```
+
+- Install the CAPI controllers using [clusterctl](https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl).
+
+```sh
+clusterctl init --infrastructure packet
+```
+
+- Set env vars with cluster config.
+
+```sh
+# Get the project ID from the project settings in the console
+export PROJECT_ID="<PROJECT_ID>"
+
+# Use Paris metro (Equinix region)
+export METRO="pa"
```

Comment:
Do we want to consider carbon intensity when deciding which metro to use?

I chose Paris since France has a low average carbon intensity and this metro has a good selection of node types.

Full list is here and we can easily change if I'm over thinking this :)
https://deploy.equinix.com/developers/docs/metal/locations/metros/

Reply:
This is a very important callout! We can hardcode Paris for now and make the testing carbon-aware in the future. Would you like to create a new issue for this to add it to our backlog? 🌟

Reply:
Yes sure I'll create an issue for it. 👍